📝 Update docs about Spaces' environment (#878)
huggingface/hub-docs: docs/hub/security-secrets.md

# Secrets Scanning

> It is important to manage [your secrets (env variables) properly](./spaces-overview#managing-secrets-and-environment-variables). The most common way people expose their secrets to outside world is hard-coding their secrets in their `app.py` files directly, which makes it possible for a malicious user to utilize your secrets and services your secrets have access to. 

Reviewer: ```suggestion
It is important to manage [your secrets (env variables) properly](./spaces-overview#managing-secrets-and-environment-variables). The most common way people expose their secrets to the outside world is by hard-coding their secrets in their `app.py` files directly, which makes it possible for a malicious user to utilize your secrets and services your secrets have access to. 
```

> For example, here is how example `app.py` described above might look like:

Reviewer: ```suggestion
For example, this is what a compromised `app.py` file might look like:
```
